app/javascript/readingList/components/TagList.jsx

```diff
@@ -0,0 +1,49 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+/**
+ *
+ * @param {object} props
+ * @param {Array<string>} props.availableTags A list of available tags.
+ * @param {string} [props.selectedTag=''] The currently selected tag.
+ * @param {function} A handler for when the selected tag changes.
+ */
+export function TagList({ availableTags, selectedTag = '', onSelectTag }) {
+  return (
+    <select
+      class="crayons-select"
+      aria-label="Filter by tag"
+      onBlur={(event) => {
+        // We need blur for a11y, but we also don't want to make the same search query twice
+        // if the tag hasn't changed since the previous onChange.
+        const { value } = event.target;
+
+        if (value === selectedTag) {
+          return;
+        }
+
+        onSelectTag(event);
+      }}
+      onChange={onSelectTag}
+    >
+      {selectedTag === '' && <option>Select a tag...</option>}
+      {availableTags.map((tag) => (
+        <option
+          selected={tag === selectedTag}
+          className={`crayons-link crayons-link--block ${
+            tag === selectedTag ? 'crayons-link--current' : ''
```

Comment:
Out of curiosity, why not this:

```
tag === selectedTag && 'crayons-link--current'
```

^ since there's no alternative value anyway..?

Reply:
If you do it this way, it will add a CSS class name called "false".

Reply:
😲 javascript is fun...